lib/rack/tracker/facebook_pixel/template/facebook_pixel.erb

```diff
@@ -18,7 +18,7 @@
 <% if events.any? %>
   <script type="text/javascript">
     <% events.each do |event| %>
-      fbq("<%= event.name %>", <%= event.write %>);
+      fbq("<%= event.name %>", <%= event.write %><%= event.with_event_id %>);
```

Comment:
Hi, thank you for the contribution. I think this should be part of `event#write`. 
Would also be more consistent with the other handlers.